modules/auxiliary/scanner/http/docker_enum.rb

```diff
@@ -0,0 +1,49 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
```

Comment:
Not necessary, but you're missing some parameters here and in the `super`.

Reply:
I think I have covered all that come in:
https://github.com/rapid7/metasploit-framework/wiki/How-to-get-started-with-writing-an-auxiliary-module

Anyhow, if you have some particular parameter please let me know

Reply:
Thanks! Addressed in bb8ac4a7ab